doc/release/RELEASE-NOTES.md

```diff
@@ -15,6 +15,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 #### Scheme changes
 
 ### Code/API changes
+* [OSDEV-1336](https://opensupplyhub.atlassian.net/browse/OSDEV-1336) - Introduced a new PATCH `/api/v1/production-locations/{os_id}/` endpoint based on the API v1 specification. This endpoint allows the creation of a new moderation event for updating the production location with the given details. Basically, the endpoint can be used to contribute to an existing location.
```

Comment:
This might be a strange question (I might have forgot something but I didn't find this in the ticket): since this endpoint will be triggered (mostly) from the SLC, will it implicitly set `source=SLC` to the request payload? Or this won't change `source` field at all?

Reply:
The source should be passed as `SLC`; otherwise, it will be set to `API`, which is the default behavior.

Reply:
> The source should be passed as SLC; 
Could you give me a reference in the codebase where this is happening?

Reply:
Sure. Here it is ->
https://github.com/opensupplyhub/open-supply-hub/blob/main/src/django/api/serializers/v1/contribution_moderation_event_source_field_serializer.py#L12